migrations/20171228151840_WpYoastIndexable.php

```diff
@@ -15,17 +15,49 @@ class WpYoastIndexable extends Ruckusing_Migration_Base {
 
 	/**
 	 * Migration up.
+	 *
+	 * @return void
 	 */
 	public function up() {
+		$this->add_table();
+	}
+
+	/**
+	 * Migration down.
+	 *
+	 * @return void
+	 */
+	public function down() {
+		$this->drop_table( $this->get_table_name() );
+	}
+
+	/**
+	 * Creates the indexable table.
+	 */
+	private function add_table() {
```

Comment:
I would move the content of this to the up method. There is no reason to have a separate method for this.

Reply:
Ah, I see there is a method for the adding the indexes as well. Maybe this is okay.